src/tools/rustfmt/tests/source/type-ascription.rs

```diff
@@ -1,10 +1,10 @@
-
```

Comment:
Would probably be easier to delete these two rustfmt files instead of using inline comments to convey changes (especially since version control will cover that history for us). Not super critical to do so, but if any other changes need to be made to this PR then would likely be worth removing these too